.babelrc

```diff
@@ -2,18 +2,9 @@
   "presets": [
     ["@babel/env", {
       "modules": "commonjs",
```

Comment:
@dfcook do you believe this ` "modules": "commonjs",` statement necessary, given the environment VTL runs on?

Reply:
No, probably not, feel free to change